eZ/Publish/API/Repository/SearchService.php

```diff
@@ -107,6 +107,13 @@ interface SearchService
      */
     public const CAPABILITY_ADVANCED_FULLTEXT = 64;
 
+    /**
+     * Capability flag for aggregation feature for use with {@see ::supports()}.
+     *
+     * @since 8.2
```

Comment:
Not sure what versioning we are using right now, but in terms of ezplatform-kernel this should be `1.2`

Reply:
Actually, this should be eZ Platform version. Thanks for spotting this.

Reply:
Fixed in 3810d17